Tests/TuistGeneratorAcceptanceTests/GenerateAcceptanceTests.swift

```diff
@@ -313,10 +313,6 @@ final class GenerateAcceptanceTestsiOSAppWithCustomScheme: TuistAcceptanceTestCa
     func test_ios_app_with_custom_scheme() async throws {
         try await setUpFixture(.iosAppWithCustomScheme)
         try await run(GenerateCommand.self)
-        try await run(BuildCommand.self)
-        try await run(BuildCommand.self, "App-Debug")
```

Comment:
In my debugging I found these made no difference, so removing them may speed up tests a little.

Reply:
We want to ensure the project is still buildable with those specific schemes, so I'd keep them.